redact/gatsby-config.js

```diff
@@ -23,11 +23,15 @@ module.exports = {
         sidebarCategories: {
           null: [
             "index",
-            "reference",
           ],
-          Redactors: [
+          "Default Redactors": [
             "database-connection-strings",
             "aws-credentials",
+            "generic-connection-strings",
+            "passwords-and-tokens",
```

Comment:
Should IP Addresses be separated into it's own item?
Can we split passwords and API tokens into 2?
I think the "generic connection strings" looks more like "http and ftp endpoints with usernames and passwords".  What name could we use to make this more specific, "generic connection strings" doesn't make sense to me here.

Reply:
This isn't what shows up in the list - this is the filepath

here's a screenshot:
![image](https://user-images.githubusercontent.com/2318911/84923996-f7aac100-b095-11ea-98c8-b7439e31752c.png)